infra/cdoc_web.Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Base image
+FROM maven:3.8.6-openjdk-11
+# Copy encryption source
+WORKDIR /app/cdoc_jar
+COPY cdoc/cdoc_jar/pom.xml pom.xml
+COPY cdoc/cdoc_jar/src src
+COPY .git .git
+
+# Package project to jar
+RUN mvn package
+
+
+# Copy webapp source
+WORKDIR /app/cdoc_web
+COPY cdoc/cdoc_web/pom.xml pom.xml
+COPY cdoc/cdoc_web/src src
+
+# Package project to jar
+RUN mvn package
```

Comment:
Compiling and running should be in 2 separate containers, look up Docker multi-stage builds